tasks/distro.js

```diff
@@ -151,14 +145,17 @@ exec('electron-builder', args, {
   stdio: 'inherit'
 });
 
-function getPublishOptions(publish, nightly, onDemand, region) {
+function getPublishOptions(publish, nightly, onDemand) {
   if (typeof publish === undefined) {
     return [];
   }
 
   if (nightly || onDemand) {
     const bucket = process.env.AWS_BUCKET;
 
+    // region has to be set explicitly to avoid permissions problems
+    const region = process.env.AWS_REGION;
```

Comment:
Related to my build-on-demand PR feedback / comment: https://github.com/camunda/camunda-modeler/pull/2226/files#r618659228 :wink:

Reply:
Indeed.